lib/g_searcher_web/controllers/report_controller.ex

```diff
@@ -29,6 +29,20 @@ defmodule GSearcherWeb.ReportController do
     end
   end
 
+  def show(conn, %{"id" => report_id}) do
+    user = conn.assigns.current_user
+
+    case Reports.get_by(%{id: report_id, user_id: user.id}) do
+      {:ok, report} ->
+        conn
```

Comment:
This is only one statement. Let's not use a pipe operator to follow our convention as well. 🙌
```elixir
render(conn, "show.html", report: report)
```